### ISSUE TITLE
Cannot stream remote RTSP on v3.0, times out with "source 0 error: EOF"

### ISSUE THREAD

**Issue by keithajmani** [AlexxIT/WebRTC#408]
On v2.31, everything works perfectly. Going to v3.0 w/ go2rtc breaks my custom cards in Lovelace. Every stream fails to start, with the error "source 0 error: EOF"

Environment:
- HomeAssistant using HASS managed (HASS image running in a VM on a Synology NAS) -- HASS has a different IP from the NAS
- Source for RTSP streams is a Scrypted rebroadcast URL (Scrypted runs in a docker container on the NAS, outside of HASS, using same IP as the NAS)
- The NAS already has a service running on port 8554 (wyze docker bridge in a docker container), but this is on a different IP than the HASS VM
- All rtsp:// URLs work with v2.31, and also work in VLC. No usernames/passwords required. Rolling back to v2.31 from v3.0 restored all functions.

Any ideas what I may be missing? Clearly it's related to to go2rtc.

**Comment by AlexxIT** [comment]
You need to increase log level to trace and check

**Comment by AlexxIT** [comment]
You need to disable certificate check. Because your invalid:
rtspx://192.168.1.1:7441/removed

**Comment by GeorgeCastanza** [comment]
Thanks for responding.  How might I disable certificate check?  I did look, but cant see certificates referenced anywhere.

**Comment by AlexxIT** [comment]
I gave you an example above

**Comment by yinzixie** [comment]
changed to `rtspx` and got new error `first record does not look like a TLS  handshake`

**Comment by keithajmani** [comment]
Finally buckled down to try and solve this today. Turns out that my source RTSP camera is the issue: I am using Scrypted's rebroadcast plugin, so that I don't have a bunch of concurrent active connections to the camera. 

I had set up one camera to use the "Scrypted / TCP"  as the RTSP parser, and this was causing the issues for go2rtc. When I changed it to use Ffmpeg/TCP, everything started work. 

Seems like an upstream issue in the Scrypted RTSP parser. Will open a ticket there.
